app/presenters/content_change_presenter.rb

```diff
@@ -0,0 +1,37 @@
+class ContentChangePresenter
+  EMAIL_DATE_FORMAT = "%I:%M %P on %-d %B %Y".freeze
+
+  def initialize(content_change:)
```

Comment:
Oh super minor, but doesn't really feel like this needs keyword arguments. We end up with calling it looking like this `ContentChangePresenter.call(content_change: content_change)` which is a lot of content changes.

Reply:
I agree. I didn't use them originally but we (@thomasleese and I) discussed making it consistent with the other workers. Happy to change back if you prefer.

Reply:
I think it'd be better back. But happy for you and @thomasleese to decide.

Reply:
I'm happy either way.

Reply:
Ok... i've fixed it in ff29953 and 3ef6219. Should be good to go now if one of you could reapprove.